src/app/Http/Controllers/SynchronizationController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use App\Enums\PritomnostAbsenceType;
+use App\Models\BusinessTrip;
+use App\Models\PritomnostAbsence;
+use App\Models\PritomnostUser;
+use App\Models\User;
+
+class SynchronizationController extends Controller
+{
+    /**
+     * Synchronize users between Cesty and Pritomnost databases.
+     *
+     * @return void
+     */
+    public function syncUsers(): void
+    {
+        /**
+         * Get all users from the database Pritomnost
+         */
+        $pritomnostUsers = PritomnostUser::all();
+
+        foreach ($pritomnostUsers as $pritomnostUser) {
+            /**
+             * Check if the user exists in the database Cesty
+             */
+            $cestyUser = User::where('personal_id', $pritomnostUser->personal_id)->first();
+
+            if ($cestyUser) {
+                /**
+                 * Update user details in the Cesty database based on the Pritomnost database
+                 */
+                $cestyUser->update([
```

Comment:
Takyto update zatial neprejde, kvoli mass assignment protection, budem to musiet povolit v tych modeloch.

Reply:
Toto by uz malo fungovat